images/win/scripts/Installers/Install-Meteor.ps1

```diff
@@ -4,3 +4,32 @@
 ################################################################################
 
 Choco-Install -PackageName meteor
+
+# Since PATH has changed, we'll reload so the current shell can use it.
+Update-SessionEnvironment
+
+# Prefetch the recent Meteor releases
+meteor update --release 2.0
+meteor update --release 1.21.12
+meteor update --release 1.11
+meteor update --release 1.10.3
+meteor update --release 1.10.2
+meteor update --release 1.10.1
+meteor update --release 1.9.3
+meteor update --release 1.9.2
+meteor update --release 1.9.1
+meteor update --release 1.9
+meteor update --release 1.8.3
+meteor update --release 1.8.2
+meteor update --release 1.8.1
+meteor update --release 1.8.0
+
+# Copy the meteor data to `All Users\AppData`
+robocopy "C:\Users\builderadmin.testbuilder1\AppData\Local\.meteor" "C:\Users\All Users\AppData\Local\.meteor" /MIR /SEC /XJD /R:5 /W:5 /MT:32 /V /NP
+
+# Update the permissions on the copied files so all users can execute and modify them
+# Since PowerShell assigns meaning to the `(` and `)`, we have to wrap the call to icacls in `Invoke-Command`
+Invoke-Expression -Command ('icacls "C:\Users\All Users\AppData\Local\.meteor" /grant All:(OI)(CI)F /t /c')
```

Comment:
What are you using Invoke-Expression here, but not above or below?

Reply:
The explanation is in the comment... PowerShell doesn't like the `(` or `)` characters. There's powershell commandlets to do the same thing, so we could also just figure out the powershell way of doing this.

Reply:
But I figured out an alternate way to invoke `icacls` that does work.